ext/reflection/php_reflection.stub.php

```diff
@@ -114,10 +114,10 @@ public function __toString(): string {}
     public function isDisabled() {}
 
     /** @return mixed */
-    public function invoke(mixed ...$args) {}
+    public function invoke(mixed ...$arguments) {}
```

Comment:
This is a BC now by https://wiki.php.net/rfc/named_params, right?

Reply:
The point is to rename everything to something consistent before PHP 8.0 hits so that there is no BC break for that.

Reply:
@nikic Would you prefer keeping the `$args` name, right?

Reply:
Yeah ^^

Reply:
OK, I'll revert these changes